pmlc/rt/xsmm.cc

```diff
@@ -34,6 +34,35 @@ extern "C" int64_t plaidml_rt_xsmm_gemm_dispatch_f32(int32_t lda, int32_t ldb,
   return reinterpret_cast<int64_t>(sgemm);
 }
 
+extern "C" void plaidml_rt_xsmm_brgemm_invoke_f32(int64_t funcAddr, float *a,
+                                                  float *b, float *c,
+                                                  int64_t lBr) {
```

Comment:
What is `lBr`? Can we use a more descriptive variable name?

Reply:
Done. I changed it to `numBatches`.